<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_widget</class>
 <widget class="QWidget" name="user_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пользователь - Blackland Radio</string>
  </property>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>*user*</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pfp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="desc">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="total_tracks">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Всего альбомов:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_albums">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="albums_table">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>360</y>
     <width>770</width>
     <height>401</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>254</number>
   </attribute>
   <column>
    <property name="text">
     <string>Название</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Жанр</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Год</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Альбомы</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
